vmdk_plugin/Makefile

```diff
@@ -350,7 +350,7 @@ test-all:  test e2e-test testremote
 # An end-to-end test target for vSphere Docker Volume Driver
 e2e-test:
 	$(log_target)
-	$(SCRIPTS)/build.sh e2e-dkrVolDriver-test "$(ESX)" "$(VM1)" "$(VM2)"
+	$(SCRIPTS)/build.sh e2e-dkrVolDriver-test "$(ESX)" "$(VM1)" "$(VM2)" "$(DS1)" "$(DS2)" "$(CNAME)" "$(T1)"
```

Comment:
I think the govc commands should be part of a script that runs these commands, parses the results and populates the env before running the automated tests. That way the user has the flexibility to change the script and run with different values or set their env with their choice of data stores and other values and run the test. It's easier to parse the govc values in script than in go code. What do you'll say.